test/snapshots.js

```diff
@@ -13,3 +13,9 @@ export function axisLeftScaleLinearNonNumericRange() {
   svg.append("g").call(axisLeft(scaleLinear().range([0, "500"])));
   return svg.node();
 }
+
+export function axisLeftTickSizeFunction() {
+  const svg = create("svg");
+  svg.append("g").call(axisLeft(scaleLinear()).tickSize(function(d, i) { return i * 2 }));
+  return svg.node();
+}
```

Comment:
```suggestion
export function axisLeftTickSizeFunction() {
  const scale = scaleLinear([0, 10], [10, 140]);
  const svg = create("svg");
  svg.append("g")
    .attr("transform", "translate(20, 0)")
    .call(axisLeft(scale).tickSize(function(d, i) { return i * 2 }));
  return svg.node();
}
```

make the snapshot readable